File: Gaussian.m
function [ array ] = Gaussian( size )
%GAUSSIAN creates a size by size array with gaussian values centred in the
%middle.

array=zeros(size,size);
x0=(size/2)+0.5;
y0=x0;

for x=1:size
    for y=1:size
        array(x,y)=exp(-((x-x0)^2+(y-y0)^2)/(2*100));
    end
end


end

File: python2/Gaussian.py
import numpy as np
def Gaussian(size):
#     np.arange(0,size)
#     np.linspace(0,size)
    sigma = size * 5 /32
    arr = np.zeros((size,size))
    y0 = x0 = (size + 1.) /2.
    for x in range(size):
        for y in range(size):
            arr[x,y] = np.exp(-(( x-x0)**2 + (y-y0)**2 )/(2*sigma**2))
    return arr
